agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -3748,8 +3748,10 @@ bool slave_thread::send_cmdu_to_controller(ieee1905_1::CmduMessageTx &cmdu_tx)
         beerocks_header->actionhdr()->radio_mac() = hostap_params.iface_mac;
         beerocks_header->actionhdr()->direction() = beerocks::BEEROCKS_DIRECTION_CONTROLLER;
     }
-    return message_com::send_cmdu(master_socket, cmdu_tx, backhaul_params.controller_bridge_mac,
-                                  backhaul_params.bridge_mac);
+
+    static const std::string MULTICAST_MAC_ADDR = "01:80:c2:00:00:13";
+    auto dst_addr = multicast ? MULTICAST_MAC_ADDR : backhaul_params.controller_bridge_mac;
```

Comment:
Since multicast is never used for VS messages, there is nothing in common between the multicast and non-multicast case. So I think it's better to create a separate function send_cmdu_multicast. (nitpick)

Reply:
not critical IMO if it is a default argument or a different function.
Anyway, when we will move to a unified Agent, we send it directly to the bus. So this function will look differently.